docs/js/features/connectivity/on-premise.md

```diff
@@ -0,0 +1,49 @@
+---
+title: On-Premise Connectivity in the SAP Cloud SDK for JavaScript
+sidebar_label: On-Premise Systems
+description: This article describes how the SDK request to On-Premise SAP S/4 HANA systems.
+keywords:
+- sap
+- cloud
+- sdk
+- On-Premise
+- connectivity
+- JavaScript
+- TypeScript
+---
+
+
+## Introduction ##
+
+This document has overlap to the [proxy handling](./proxy.md), [destination lookup](./destination.md) and [On-Premise connectivity from the BAS](../../guides/bas-external-system.md).
+The focus of this document is on the steps necessary to connect to an On-Premise system and how the SDK implements these steps.
+The crucial problem with On-Premise SAP S/4 HANA systems is, that they are not exposed to the internet.
```

Comment:
These first lines sound more scientific than I would expect in an online SDK documentation. Maybe you can motivate it the other way around and leave out "the focus of this document". This should become clear from the description.

```suggestion
The crucial problem with SAP S/4HANA On-Premise systems is, that they are not exposed to the internet.
The SAP Cloud SDK has some useful features that allow you to connect to On-Premise systems.
```

(This sounds a bit dry, but I guess you get what I mean).

Reply:
i adjusted the sentences and added a picture